scipy/stats/stats.py

```diff
@@ -2331,6 +2333,94 @@ def zmap(scores, compare, axis=0, ddof=0):
         return (scores - mns) / sstd
 
 
+def gstd(a, axis=0, ddof=1):
+    """Calculate the geometric standard deviation of an array
+
+    The geometric standard deviations is defined as the exponent of the
+    standard deviation of ``log(a)``. Mathematically the population
+    geometric standard deviation is evaluated as::
+
+        gstd = exp(std(log(a))
+
+    The geometric standard deviation describes the spread of a set of numbers
+    where `gmean` is the preferred mean. It is a multiplicative factor, and
+    therefore a dimensionless quantity.
+
+    Parameters
+    ----------
+    a : array_like
+        An array like object containing the sample data.
+    axis : int or None, optional
+        Axis along which to operate. Default is 0. If None, compute over
+        the whole array `a`.
+    ddof : int, optional
+        Degrees of freedom correction in the calculation of the
+        geometric standard deviation. Default is 1.
+
+    Returns
+    -------
+    ndarray or float
+        An array of the geometric standard deviation. If `axis` is `1` or `None`
```

Comment:
Suggest to remove backticks around None / 1 to be consistent with notation above

Same in mstats